src/evict/evict_lru.c

```diff
@@ -2300,6 +2300,10 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
             if (ret == WT_ROLLBACK) {
                 --cache->evict_aggressive_score;
                 WT_STAT_CONN_INCR(session, txn_fail_cache);
+                if (timer)
+                    WT_IGNORE_RET(__wt_msg(session, "%s",
+                      "Application thread returned a rollback error because "
+                      "it was forced to evict and eviction is stuck"));
```

Comment:
How about :

```WT_IGNORE_RET(__wt_msg(session,
                      "Application thread returned a rollback error because "
                      "it was forced to evict and eviction is stuck"));```

Reply:
Fixed.